test/fixtures/organization_admin_user.js

```diff
@@ -8,5 +8,11 @@ module.exports = {
   // hash of 'youpin_admin' password
   password: '$2a$10$iorOMFOPboPeF20W20DKruey2UXXa4eOQSuReOMlxXnqNe5t6Egaq',
   email: 'organization_admin@youpin.city',
+  organization_and_role_pairs: [
+    { organization: '579334c74443625d6281b6dd', role: ORGANIZATION_ADMIN },
+  ],
+  organization_and_department_pairs: [
```

Comment:
Oh, got it. That org_admin guy must be in `57933111556362511181ccc1` not `57933111556362511181bbb1`.